src/components/apply/ApplicantForm.js

```diff
@@ -20,25 +22,42 @@ const headingStyle = {
 
 const SectionHeading = ({ children }) => <div style={headingStyle}>{children}</div>;
 
-const ApplicantForm = ({ handleSubmit }) => (
+const ApplicantForm = ({ handleSubmit, addJointApplicant, history }) => (
   <div style={{marginBottom: 12}}>
-    <form onSubmit={handleSubmit((values) => console.log(values))} style={{display:'flex', flexDirection: 'column', alignItems: 'center'}}>
-      <SectionHeading>Your Identity</SectionHeading>
-      <IdentificationForm />
-      <SectionHeading>Contact Information and Address</SectionHeading>
-      <ContactInfo />
-      <Flex alignSelf='center' width='45%' style={{marginLeft: 80, marginBottom: 0, marginTop: 30}}>
-        <Field
-            name="addJointApplicant"
-            component={Checkbox}
-            label="Add joint applicant to my membership application"
-            labelStyle={{color: colors.basic}}
-            iconStyle={{fill: colors.basic, marginLeft: 10}}
-        />
-      </Flex>
-      <center><ContinueButton title='KEEP GOING' buttonProps={{type: 'submit'}} style={{ marginTop: 10 }}/></center>
-    </form>
-  </div>
+    <form onSubmit={handleSubmit((values) =>{
```

Comment:
Imagine that we want to submit to the backend at this point to allow people to save their application and resume. Or imagine that we want to record some analytics about how long it took the user to fill out our form. Or imagine that we need to do an identity verification.

Basically, if this turns into an asynchronous process in anyway, it becomes really beneficially to use some of the tools the redux ecosystem offers us for handling those.

Not saying this needs to be converted to that. As long as it is pulled out into a named function I will approve it. But it is something to think about.